synapse/axon.py

```diff
@@ -169,6 +236,10 @@ async def _initBlobStor(self):
         self.blobs = self.blobslab.initdb('blobs')
         self.onfini(self.blobslab.fini)
 
+    def _initAxonHttpApi(self):
+        self.addHttpApi('/api/v1/axon/files', AxonFilesHttpV1, {'cell': self})
+        self.addHttpApi('/api/v1/axon/files/(.*)', AxonFileHttpV1, {'cell': self})
```

Comment:
similarly `api/axon/v1/files/by/sha256/` and we could tighten down the regex too :)

Reply:
Updated.